### ISSUE TITLE
Number of songs in playlist limited ?

### ISSUE THREAD

**Issue by RRRickkk** [pascalopitz/unoffical-sonos-controller-for-linux#171]
Good day Pascal !

First of all, thanks for this excellent Sonos alternative for linux. Also, Happy Holidays to you and yours !

I was wondering if there was a limit in UnSoCo for the number of songs in a playlist. I have one with 1,093 songs that does not appear completely in UnSoCo.

Thanks,

Rick

**Comment by pascalopitz** [comment]
Playlist being Sonos Playlist, Service Playlist or Queue?

**Comment by RRRickkk** [comment]
Hey Pascal,

I am referring to Sonos playlists.

**Comment by pascalopitz** [comment]
Nope, not limited. The playlist is stored on the Sonos device itself. Scrolling to the end of the list should load another batch and append entries, like all other browser list views.

Is there maybe an error in the console that gives any clues? Can you export a state dump and append as file?

**Comment by RRRickkk** [comment]
Hi again Pascal,

There are quite a few errors and warning in the console ... I am attaching 2 state dump files exported after trying to scroll to the end of the playlist as well as an app settings export. Thanks again !

[unoffical-sonos-controller.appState(2).json.txt](https://github.com/pascalopitz/unoffical-sonos-controller-for-linux/files/7784712/unoffical-sonos-controller.appState.2.json.txt)
[unoffical-sonos-controller.appState.json.txt](https://github.com/pascalopitz/unoffical-sonos-controller-for-linux/files/7784714/unoffical-sonos-controller.appState.json.txt)
[unoffical-sonos-controller.settings.json.txt](https://github.com/pascalopitz/unoffical-sonos-controller-for-linux/files/7784721/unoffical-sonos-controller.settings.json.txt)
!

**Comment by RRRickkk** [comment]
Pascal : if this can be at all helpful - When I scroll down the playlist to where it stops loading and play that last song appearing in the list, it plays normally. I can also advance to the next songs in the list, even though they do not appear (show) in the list "box" itself.

Also, when scrolling to where the list stops refreshing, the console starts to show a whole pile of "404 not found" errors, even though the files are there (and play).

Thanks again.

Rick

**Comment by pascalopitz** [comment]
Curious. I am not sure why. I can see it loaded 1000 tracks. Maybe a limitation with node-sonos?
